src/clusterfuzz/_internal/platforms/android/symbols_downloader.py

```diff
@@ -141,6 +137,13 @@ def download_system_symbols_if_needed(symbols_directory):
 
   build_params_check_path = os.path.join(symbols_directory,
```

Comment:
Can you add a helper for these rather than duplicate code?

i.e. roughly:

```python
def check_symbols_cached(symbols_directory, build_params):
  build_params_check_path = os.path.join(symbols_directory, ...)
  cached_build_params = ...
  return cached_build_params and cached_build_params == build_params)
```

Reply:
Done